policies/committer-policy.md

```diff
@@ -0,0 +1,102 @@
+
+# Policy for OpenSSL Committers
+
+## Who is a committer?
+
+OpenSSL committers are contributors who have commit access to the OpenSSL
+source code repository.  Committers review and commit their own patches
+as well as those of other contributors.
+
+## How to become a committer?
+
+Commit access is granted by the [OMC] typically on the recommendation
+of the [OTC] (see the [OpenSSL Bylaws]).
```

Comment:
This is slightly confusing. The bylaws say:

"Commit access is granted by invitation from the OTC and requires a prior OMC vote of acceptance."

So the bylaws say "Commit access is granted by...the OTC", but this policy says its granted by the OMC.

Reply:
It's the OMC voting acceptance, they are granting.
The OTC recommends.